ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml

```diff
@@ -32,6 +32,8 @@ chain:
           echo "$(date --rfc-3339=seconds) Clusteroperators not yet ready"
           sleep 1s
       done
+      # Data for cluster bot.
+      echo "https://$(oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
```

Comment:
just for future consideration... we should likely have this in HostedCluster status